docs/transformers.ipynb

```diff
@@ -9,7 +9,7 @@
    },
```

Comment:
<p>nit: "which composes a couple of the available" --&gt; use few instead of couple, since we are composing more than 2 transformers. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>